cogs/ask_trivia_questions.py

```diff
@@ -59,11 +118,11 @@ async def ask_trivia_question(
 
         embed = discord.Embed(
             title=f"Trivia Question!",
-            description=f"{question_txt} \n\n Difficulty: {question_diff} \n\n Incorrect answers so far: {self.incorrect_answers}",
+            description=f"{question_txt} \n\n Difficulty: {question_diff} \n\n Category: {selected_category.name.title().replace('_', ' ')} \n\n Incorrect answers so far: {self.incorrect_answers}",
```

Comment:
Add the category to the embed messages for a correct/incorrect answer as well, as it is currently not visible after the user guesses: 

![image](https://github.com/unswwit/discord-bot/assets/101443389/894cdf7f-552a-432b-9b93-29963a6cf74d)